README.md

```diff
@@ -132,6 +132,98 @@ In order to support atomify turtles all the way down, you can also specify your
 
 For detailed information on configuring Rework plugins in package.json see the [relevant section](https://github.com/Techwraith/atomify-css#packagejson-config) of the atomify-css README.
 
+## Mini Tutorial
+### How to build a re-usable component
+(__Warning:__ _The following example is only interesting from a technical and not at all from an esthetical perspective)_
+1. __Scaffold component project structure__
+  * `componentA/`
```

Comment:
Yes, i think my editors (atom.io) markup preview plugin messed things up quite a bit.
I'll soon do another push to fix it.